lib/cluster-plugin.js

```diff
@@ -26,12 +26,14 @@ module.exports = function(clusterConfig) {
             if (!package.happner) return;
             if (!package.happner.dependencies) return;
 
-            if (package.happner.dependencies["$broker"]) {
+            if (package.happner.dependencies['$broker']) {
+              mesh._mesh.config.brokered = true;
```

Comment:
Where is this flag been used?  Was this just not been set?

Reply:
This is being used in happner-2 describe method - it flags the description as being brokered, so when it is passed around the cluster via happner-client - brokered descriptions are ignored.